app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb

```diff
@@ -176,6 +184,20 @@ def validate_ip_address(_field, _values, _dlg, _fld, value)
     return _('IP-address field has to be either blank or a valid IPv4 address') unless valid
   end
 
+  def validate_placement_group(_filed, _values, _dlg, _fld, value)
+    return if value.blank?
+
+    placement_group = ar_ems.placement_groups.find_by(:ems_ref => value)
```

Comment:
Recommend either `.find_by!` which will raise an exception if it can't find the record or handling `placement_group` being `nil` if it isn't found by `value`